rpm/spec.in

```diff
@@ -81,7 +86,7 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_docdir}/wolfssl/README.txt
 %{_docdir}/wolfssl/QUIC.md
 
-%{_libdir}/libwolfssl@LIBSUFFIX@.so*
+%{_libdir}/libwolfssl@LIBSUFFIX@.so.*
```

Comment:
These changes here to "so" and "libdir" are correct, there was another recent report about it.